app/policies/ssl_redirect_exclusion_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "set"
+
+class SslRedirectExclusionPolicy
+  EXEMPT_PATH_PATTERNS = Set.new([
+    %r{^/api/metadata$},
+    %r{^/health-check$},
+    %r{^/idt/api/v1/},
+    %r{^/api/docs/v3/},
+    %r{^/idt/api/v2/}
+                                 ]).freeze
```

Comment:
Looks like [codeclimate is still being a menace](https://codeclimate.com/github/department-of-veterans-affairs/caseflow/pull/19328) 😆

Have you tried just making this an `Array` to get around the indentation issues? 
Given there are only 5 elements here, I don't think there will be any meaningful performance differences between accessing a `Set` versus an `Array`.

```suggestion
class SslRedirectExclusionPolicy
  EXEMPT_PATH_PATTERNS = [
    %r{^/api/metadata$},
    %r{^/health-check$},
    %r{^/idt/api/v1/},
    %r{^/api/docs/v3/},
    %r{^/idt/api/v2/}
  ].freeze
```